src/SingleSelectOption.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { colors, spacers } from './theme.js'
+
+const SingleSelectOption = ({
+    label,
+    active,
+    disabled,
+    onClick,
+    className,
+}) => (
+    <a
+        className={cx(className, {
+            disabled,
+            active,
+        })}
+        onClick={onClick}
+    >
+        {label}
+
+        <style jsx>{`
+            a {
+                display: block;
+                cursor: pointer;
+                font-size: 14px;
+                text-decoration: none;
+                color: ${colors.grey900};
+                padding: ${spacers.dp8} ${spacers.dp12};
+                white-space: nowrap;
+                overflow: hidden;
+                text-overflow: ellipsis;
+            }
+
+            a:hover {
+                background-color: ${colors.grey200};
+            }
+
+            a:active,
+            a.active {
+                background-color: ${colors.grey400};
+                cursor: auto;
+            }
+
+            a.disabled {
+                color: rgba(0, 0, 0, 0.3);
+                cursor: not-allowed;
```

Comment:
This cursor style is not working for me, is that because of Storybook perhaps?
Firefox on Mac

Reply:
We've hit this before, and in other components we wrap the event handler and check if `props.disabled` is false, and only then do we call the event handler that was passed in as a callback.

Reply:
Fixed in d4cbc68be21507cb38d4aa64412a71209bc90417